client/state/push-notifications/README.md

```diff
@@ -0,0 +1,46 @@
+PushNotifications
```

Comment:
This needs to be rewritten & the README in `client/lib/push-notifications/` should be updated to only describe the service worker in that dir.